docs/spherical-video-v2-rfc.md

```diff
@@ -222,12 +241,14 @@ monoscopic equirectangular video:
           [avc1: Advance Video Coding Box]
             [pasp: Pixel Aspect Ratio Box]
               ...
+            [st3d: Stereoscopic 3D Video Box]
+              [mo3d: Stereocopic mode]
```

Comment:
This line should be removed and the "unsigned int(8) stereo_mode;" line should be replaced with "stereo_mode = 0".